initfiles/componentfiles/config2xml/hardware.xsd

```diff
@@ -8,8 +8,7 @@
       <xs:element name="Hardware" hpcc:docid="????" hpcc:doc="Section to describe the hardware that composes the environment">
         <xs:complexType>
           <xs:sequence>
-            <xs:element name="ComputerType" maxOccurs="unbounded" hpcc:displayName="Defined computer types"  hpcc:class="valueSet"
-                        hpcc:doc="Defines a generalized type of computer that is used in the environment">
+            <xs:element name="ComputerType" maxOccurs="unbounded" hpcc:displayName="Defined computer types"  hpcc:class="valueSet">
```

Comment:
Why do you remove the 'hpcc:doc'?

Reply:
Not sure what you are asking. The docid for the doc team is now a custom namespace attribute (hpcc:docid"), but I don't know what it is right now. I was going to fix it when I worked on the hardware section. As far as hpcc:doc, it was something in the current XSD files. I retained it in case it was needed. It will be addressed when the hardware section is done.

Reply:
I am asking why you remove the 'hpcc:doc="Defines a generalized ..."' from the old code.

Reply:
Oh, it was removed because it was something I added a while ago while doing a first pass conversion of the XSDs. I realized that there is no doc level comments at the location or level where it was placed, so I removed it. In fact, it was not even being parsed.

Reply:
Ok